src/System/Wlog/LoggerName.hs

```diff
@@ -5,20 +5,22 @@ module System.Wlog.LoggerName
        , loggerNameF
        ) where
 
-import           Universum
+import Universum
 
-import           Data.Hashable       (Hashable)
-import           Data.Semigroup      (Semigroup)
-import qualified Data.Semigroup      as Semigroup
-import           Data.String         (IsString)
-import           Data.Text.Buildable (Buildable)
+import Data.Aeson.Types (ToJSON, ToJSONKey (..), toJSONKeyText)
+import Data.Hashable (Hashable)
+import Data.Semigroup (Semigroup)
+import Data.String (IsString)
+import Data.Text.Buildable (Buildable)
+import Formatting (Format, bprint, build, stext)
+
+import qualified Data.Semigroup as Semigroup
```

Comment:
I just noticed that this import is redundant because `universum-1.0.0` exports `<>` from `Semigroup`. I just forgot to remove it while I was migrating to `universum-1.0.0`. Could you please remove it in one of your future commits in this PR?